WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1271,13 +1271,16 @@ decisions = {
 		potential = {
 			has_dlc = "Sons of Abraham"
 			is_playable = yes
-			religion_group = jewish_group
+			OR = {
+				religion_group = jewish_group
+				religion = ebionites
+			}
 			k_israel = {
 				has_holder = no
 			}
 		}
 		allow = {
-			completely_controls = k_oriens
+			completely_controls = d_iudea # K_ORIENS IS TOO BIG, THIS IS WHAT ISRAEL HAD IN RL
```

Comment:
No. You are dead wrong. That was the actual size of Israel.

That is the ACTUAL and HISTORICAL size of the KINGDOM OF ISRAEL. To restore Israel, you must own ISRAEL'S OLD LANDS as a Jew.

Anything else is ahistorical. Period.